src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -36,6 +36,8 @@ interface SavedAddressesProps {
   commitMutation?: CommitMutation
   relay: RelayRefetchProp
   addressCount?: number
+  showModal?: boolean
```

Comment:
Is this needed? I don't see it referenced anywhere in this file.

Reply:
Sorry, my mistake. Done